### DISCUSSION TITLE
when i connect postal to mautic i get this error.

### DISCUSSION THREAD

**Discussion by winsta** [postalserver/postal#1459] [Help with using Postal]
Connection could not be established with host domain.com:stream_socket_client(): unable to connect to domain.com:25 (Connection timed out) Log data: ++ Starting Swift_SmtpTransport !! Connection could not be established with host domain.com :stream_socket_client(): unable to connect to domain.com:25 (Connection timed out) (code: 0)
++ Starting Swift_SmtpTransport !! Connection could not be established with host domain.com :stream_socket_client(): unable to connect to  @domain.com:25 (Connection timed out) (code: 0)

@willpower232

**Comment by rccess** [comment]
check the Server firewalls on which Mautic installed 
try with disable firewall 

hope this will help

**Comment by winsta** [comment]
It installed in Cpanel how can I Disable firewall

**Comment by rccess** [comment]
contact your hosting provider for the same or relay the emails of postal to 2525, most of the providers don't block 2525

**Comment by willpower232** [comment]
You should double check `postal status` and `netstat -lnp` to confirm Postal is running and is on port 25 but yes it could be a firewall issue.
